pkg/virt-controller/watch/drain/disruptionbudget/disruptionbudget_test.go

```diff
@@ -192,19 +192,22 @@ var _ = Describe("Disruptionbudget", func() {
 
 	Context("A VirtualMachineInstance given which wants to live-migrate on evictions", func() {
 
-		It("should do nothing, if a pdb exists", func() {
+		DescribeTable("should do nothing, if a pdb exists", func(evictionStrategy v1.EvictionStrategy) {
```

Comment:
Given that this `Context` is now for VMIs with `LiveMigrate` or `External` you should change the context description.

Reply:
fixed